### ISSUE TITLE
Allow overriding task data and taskrc file on the command line

### ISSUE THREAD

**Issue by fdela** [vit-project/vit#258]
Currently, taskrc file location is overridable in vit `config.ini` file (and indirectly via TASKRC envvar)

``` 
[taskwarrior]

# Full path to the Taskwarrior configuration file. Tilde will be expanded to the user's home directory.
# NOTE: This setting is overridden by the TASKRC environment variable.
#taskrc = ~/.taskrc
```
TaskWarrior gives two ways to override config file and data location, via command-line or environment:
|        | Config file | Data location |
|----  | ---             | ---                  |
| **Command-line<br>argument** | rc:_config\_file_ | rc.data.location:_data\_directory_ |
| **Environment<br>variable** | TASKRC=_config\_file_   | TASKDATA=_data\_directory_ |

See [_CONFIGURATION FILE AND OVERRIDE OPTIONS_ section in task(1) manpage](https://manpages.ubuntu.com/manpages/focal/en/man1/task.1.html#configuration%20file%20and%20override%20options) for details.

Overriding manually the environment via TASKRC and TASKDATA (used by TaskWarrior) before launching vit works currently, and shown data/used config are correct.

What would be useful it to be able to specify those via `vit` command line, as `task` does: 
> task rc.data.location:_alternateDataLocationDirectory_ rc:_alternateTaskrcConfFile_ _..._

This would allow for instance to have _completely_ separate environments, without having to rely on contexts (where it's a soft separation).

Linked issues: #257 and #235

**Comment by thehunmonkgroup** [comment]
See https://github.com/scottkosty/vit/issues/257 for the suggestion there, perhaps a single ```--default-args``` switch?

**Comment by fdela** [comment]
> See #257 for the suggestion there, perhaps a single `--default-args` switch?

Yes, a `--default-args` would do, to pass to every taskwarrior command.
Care should be taken to respect precedence of options, so they are applied by increasing precedence: `config.ini` -> environment variable -> command line option

E.g. taskrc option in config.ini -> TASKRC -> `--default-args rc:...`

**Comment by thehunmonkgroup** [comment]
I've abandoned the idea of a `--default-args` approach, too complicated.

I've also abandoned the idea of any VIT-specific overrides for task data, see https://github.com/vit-project/vit/issues/257#issuecomment-691721253 for more.

I'd still consider adding a CLI arg that would allow overriding the task data location -- this is reasonably straightforward, compliments the existing config file option, and allows for more flexibility for those using more complex setups.
